src/Symfony/Component/Notifier/Bridge/OrangeSms/OrangeSmsTransport.php

```diff
@@ -0,0 +1,117 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Notifier\Bridge\OrangeSms;
+
+use Symfony\Component\Notifier\Message\SmsMessage;
+use Symfony\Component\Notifier\Message\SentMessage;
+use Symfony\Contracts\HttpClient\HttpClientInterface;
+use Symfony\Component\Notifier\Exception\LogicException;
+use Symfony\Component\Notifier\Message\MessageInterface;
+use Symfony\Component\Notifier\Transport\AbstractTransport;
+use Symfony\Component\Notifier\Exception\TransportException;
+use Symfony\Contracts\EventDispatcher\EventDispatcherInterface;
+use Symfony\Component\Notifier\Exception\UnsupportedMessageTypeException;
+
+final class OrangeSmsTransport extends AbstractTransport
+{
+    protected const HOST = 'api.orange.com';
+
+    private $clientID;
+    private $clientSecret;
+    private $from;
+    private $senderName;
+
+    public function __construct(string $clientID, string $clientSecret, string $from, ?string $senderName, HttpClientInterface $client = null, EventDispatcherInterface $dispatcher = null)
+    {
+        $this->clientID = $clientID;
+        $this->clientSecret = $clientSecret;
+        $this->from = $from;
+        $this->senderName = $senderName;
+
+        parent::__construct($client, $dispatcher);
+    }
```

Comment:
```suggestion
Since 6.1 is PHP 8 only, why not just use constructor property promotion?
```suggestion
    public function __construct(private string $clientID, private string $clientSecret, private string $from, private ?string $senderName, HttpClientInterface $client = null, EventDispatcherInterface $dispatcher = null)
    {
        parent::__construct($client, $dispatcher);
    }
```

Reply:
Not sure Symfony uses it

Cc @symfony/mergers

Reply:
I don't see any reason not to use it in new code. But before, we have to be sure that PHP-CS-Fixer/fabbot is compatible.